python/pyspark/sql/connect/plan.py

```diff
@@ -108,7 +108,7 @@ def __init__(self, table_name: str) -> None:
 
     def plan(self, session: Optional["RemoteSparkSession"]) -> proto.Relation:
         plan = proto.Relation()
-        plan.read.named_table.parts.extend(self.table_name.split("."))
+        plan.read.named_table.unparsed_identifier = self.table_name
```

Comment:
I'm not sure this is the only place where this is used like that, I know this is the lack of test coverage, but I'd suggest to search for `split(".")`.

Reply:
```❯ rg 'split\(' python/pyspark/sql/connect/
python/pyspark/sql/connect/plan.py
111:        plan.read.named_table.parts.extend(self.table_name.split("."))
160:            else self.unresolved_attr(*(c.split(".")))

python/pyspark/sql/connect/column.py
98:        return ColumnRef(*name.split("."))
```